pubspec.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   string_scanner: ">=0.1.5 <2.0.0"
   stack_trace: ">=0.9.0 <2.0.0"
   tuple: "^1.0.0"
+  package_resolver: ^1.0.2
```

Comment:
Always declare the lowest version that has all the APIs you need. In this case, that's 1.0.0.